### PR TITLE
Feature Request #2767 Raid Sent Off Event

### PR DESCRIPTION
**Pull request by karrbs**
### Description of the Change
Added Raid Sent off event to the event list. 
Created accompanying variable changes and activity feed changes 

### Applicable Issues
#2767 

### Testing
Manual:
Raid Sent Off:

1. Add Event for Raid Sent Off 
2. Raid a given channel 
3. Once raid is completed effects are fired off and message appears in activity feed

Regression:
Incoming raid

1. Add Event for Incoming Raid
2. Receive a Raid from a different channel
3. Once raid is completed effects are fired off and message appears in activity feed

### Screenshots
activity feed
image

Event Names
image

Filter Hint Message
image

### CODE REVIEW

**Comment by dennisrijsdijk** [comment]
I feel like "Raid Sent Off" could be a very confusing event name. Something like "Outgoing Raid Completed" may be more verbose and would fit better with the new naming scheme of "Incoming Raid".

File: src/backend/events/builtin/twitch-event-source.js [hunk 1]
```diff
@@ -21,7 +21,36 @@ module.exports = {
...
+                    return `**${eventData.userDisplayName}${
+                        showUserIdName ? ` (${eventData.username})` : ""
+                    }** raided with **${eventData.viewerCount}** viewer(s)`;
+                }
+            }
+        },
+        {
+            id: "raid-sent-off",
+            name: "Raid Sent Off",
+            description: "When your raid finishes sending off",
```
Comment (phroggster) [inline src/backend/events/builtin/twitch-event-source.js:33]: End the description with a period to match existing convention:
simulate raid

File: src/backend/events/filters/builtin/raid-viewer-count.js [hunk 2]
```diff
@@ -7,7 +7,8 @@ module.exports = {
     name: "Raid Viewer Count",
     description: "Filter by how many viewers have been brought over by the raid.",
```
Comment (phroggster) [inline src/backend/events/filters/builtin/raid-viewer-count.js:8]: The description needs a smidge of polish to reflect the new bidirectionality of the raid in/out events where it's used.
filter text

**Review by phroggster — COMMENTED** [review]
Looks great, and will make a bunch of people happy!

File: src/backend/events/filters/builtin/raid-viewer-count.js (hunk 2 above)
Comment (phroggster) [reply src/backend/events/filters/builtin/raid-viewer-count.js:8]: As far as actual verbiage, "Filter by how many users have been redistributed amongst the glorious interwebs by this raid." just seems like a poor choice, but is about the best I can come up with. I can only really point out that "brought over" in this context implies to me an incoming viewer count, while the filter is actually working upon the outgoing viewer count in this instance. Someone hire a copy-writer, stat!

**Comment by phroggster** [comment]
Oh, I'm also going to suggest changing the activity feed icons for **both** the raid in and raid out events. They're currently "fad fa-siren-on" for both:
fad fa-siren

"fad fa-inbox-out" and "fad fa-inbox-in" would probably be my choices, but here's several interesting pairs that might be suitable for the job, with the images lazily screenshotted from fontawesome's 5.15.4 preview window:
| Inbound | Inbound Image | Outbound | Outbound Image |
|---|---|---|---|
| fad fa-inbox-in | fad fa-inbox-in | fad fa-inbox-out | fad fa-inbox-out |
| fad fa-sign-in-alt | fad fa-sign-in-alt | fad fa-sign-out-alt | fad fa-sign-out-alt |
| fad fa-sign-in | fad fa-sign-in | fad fa-sign-out | fad fa-sign-out |
| fad fa-portal-enter | fad fa-portal-enter | fad fa-portal-exit | fad fa-portal-exit |

**Comment by karrbs** [comment]
Screenshots
activity feed:
image

Event Names
image

Filter Hint Message
image

**Review by ebiggz — APPROVED** [review]
Looks great! Thanks for working on this!

**Pull request merged**